scripts/apps/web-publisher/controllers/WebPublisherManagerController.js

```diff
@@ -7,24 +7,151 @@
  * @requires https://docs.angularjs.org/api/ng/type/$rootScope.Scope $scope
  * @description WebPublisherManagerController holds a set of functions used for web publisher
  */
+
```

Comment:
Why the blank line? Makes it look a bit like the documentation doesn't belong to what is below it imo. But I don't feel strongly about it.